### ISSUE TITLE
Android ./gradlew assembleRelease fails

### ISSUE THREAD

**Issue by TCGXR** [nativewind/nativewind#1167]
"nativewind": "^4.1.21",
"tailwindcss": "^3.4.14"

> Task :app:createBundleReleaseJsAndAssets
debug Reading Metro config from C:\Dev\tcgxr_app\react_app\metro.config.js
warning: the transform cache was reset.
                Welcome to Metro v0.80.12
              Fast - Scalable - Integrated

error SHA-1 for file C:\Dev\tcgxr_app\react_app\node_modules\react-native-css-interop\.cache/android.js is not computed.
         Potential causes:
           1) You have symlinks in your project - watchman does not follow symlinks.
           2) Check `blockList` in your metro.config.js and make sure it isn't excluding the file path.
ReferenceError: SHA-1 for file C:\Dev\tcgxr_app\react_app\node_modules\react-native-css-interop\.cache/android.js is not computed.
         Potential causes:
           1) You have symlinks in your project - watchman does not follow symlinks.
           2) Check `blockList` in your metro.config.js and make sure it isn't excluding the file path.
    at DependencyGraph.getSha1 (C:\Dev\tcgxr_app\react_app\node_modules\metro\src\node-haste\DependencyGraph.js:179:13)
    at Transformer._getSha1 (C:\Dev\tcgxr_app\react_app\node_modules\metro\src\Bundler.js:15:26)
    at Transformer.transformFile (C:\Dev\tcgxr_app\react_app\node_modules\metro\src\DeltaBundler\Transformer.js:88:19)
    at Bundler.transformFile (C:\Dev\tcgxr_app\react_app\node_modules\metro\src\Bundler.js:43:30)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

**Comment by marklawlor** [comment]
Please provide your code via a GitHub link. I will not download random files

**Comment by TCGXR** [comment]
https://github.com/TCGXR-ORG/ERROR_1167.git

Works with connected device and `npx react-native run-android --verbose` not with `./gradlew assembleRelease`

I removed keystore etc.

**Comment by TCGXR** [comment]
The error comes from import "./global.css" in App.tsx

**Comment by TCGXR** [comment]
Hi,
same here:

windows, react native without Expo
"nativewind": "^4.1.21",
"tailwindcss": "^3.4.14"

https://github.com/TCGXR-ORG/ERROR_1167.git

    Task :app:createBundleReleaseJsAndAssets
    debug Reading Metro config from C:\Dev\tcgxr_app\react_app\metro.config.js
    warning: the transform cache was reset.
    Welcome to Metro v0.80.12
    Fast - Scalable - Integrated

error SHA-1 for file C:\Dev\tcgxr_app\react_app\node_modules\react-native-css-interop.cache/android.js is not computed.
Potential causes:
1) You have symlinks in your project - watchman does not follow symlinks.
2) Check blockList in your metro.config.js and make sure it isn't excluding the file path.
ReferenceError: SHA-1 for file C:\Dev\tcgxr_app\react_app\node_modules\react-native-css-interop.cache/android.js is not computed.
Potential causes:
1) You have symlinks in your project - watchman does not follow symlinks.
2) Check blockList in your metro.config.js and make sure it isn't excluding the file path.
at DependencyGraph.getSha1 (C:\Dev\tcgxr_app\react_app\node_modules\metro\src\node-haste\DependencyGraph.js:179:13)
at Transformer._getSha1 (C:\Dev\tcgxr_app\react_app\node_modules\metro\src\Bundler.js:15:26)
at Transformer.transformFile (C:\Dev\tcgxr_app\react_app\node_modules\metro\src\DeltaBundler\Transformer.js:88:19)
at Bundler.transformFile (C:\Dev\tcgxr_app\react_app\node_modules\metro\src\Bundler.js:43:30)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

**Comment by b1ngx** [comment]
Same error with React Native 0.76

```
error SHA-1 for file \node_modules\.pnpm\react-native-css-interop@0.1.20_react-native-reanimated@3.16.1_@babel+core@7.26.0_react-nativ_ddsrfdkumypzrhdrk72a44575i\node_modules\react-native-css-interop\.cache/android.js is not computed.
         Potential causes:
           1) You have symlinks in your project - watchman does not follow symlinks.
           2) Check `blockList` in your metro.config.js and make sure it isn't excluding the file path.
ReferenceError: SHA-1 for file \node_modules\.pnpm\react-native-css-interop@0.1.20_react-native-reanimated@3.16.1_@babel+core@7.26.0_react-nativ_ddsrfdkumypzrhdrk72a44575i\node_modules\react-native-css-interop\.cache/android.js is not computed.
         Potential causes:
           1) You have symlinks in your project - watchman does not follow symlinks.
           2) Check `blockList` in your metro.config.js and make sure it isn't excluding the file path.
    at DependencyGraph.getSha1 (\node_modules\.pnpm\metro@0.81.0\node_modules\metro\src\node-haste\DependencyGraph.js:181:13)
    at Transformer._getSha1 (\node_modules\.pnpm\metro@0.81.0\node_modules\metro\src\Bundler.js:15:26)
    at Transformer.transformFile (\node_modules\.pnpm\metro@0.81.0\node_modules\metro\src\DeltaBundler\Transformer.js:92:19)
    at Bundler.transformFile (\node_modules\.pnpm\metro@0.81.0\node_modules\metro\src\Bundler.js:43:30)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.transform (\node_modules\.pnpm\metro@0.81.0\node_modules\metro\src\lib\transformHelpers.js:134:12)
    at async transform (\node_modules\.pnpm\metro@0.81.0\node_modules\metro\src\DeltaBundler\Graph.js:164:26)
    at async visit (\node_modules\.pnpm\metro@0.81.0\node_modules\metro\src\DeltaBundler\buildSubgraph.js:82:29)
    at async Promise.all (index 3)
    at async visit (\node_modules\.pnpm\metro@0.81.0\node_modules\metro\src\DeltaBundler\buildSubgraph.js:92:5)

> Task :app:createBundleReleaseJsAndAssets FAILED
Execution failed for task ':app:createBundleReleaseJsAndAssets'.
> Process 'command 'cmd'' finished with non-zero exit value 1
```

**Comment by bpn2k4** [comment]
i am facing same problem when upgrade from v2 to v4, i can build release when using v2 but can not build release when use v4.

**Comment by usmanmughalji** [comment]
I am facing same issue.

**Comment by marklawlor** [comment]
Please do not post "me too" comments without adding any extra information. You just add noise and delay a solution.

@TCGXR Your reproduction is missing an `android` folder. I cannot use your reproduciton
@b1ngx I cloned your repro and ran `./gradlew assembleRelease` successfully without issue. Please try clearing your Metro cache or provide more information about your setup.

**Comment by bpn2k4** [comment]
Hi, i cloned @b1ngx repo [https://github.com/b1ngx/nativewind-issue.git](b1ngx/nativewind-issue) and encountered this error when using the bundle command.
OS: window 11
Node: v21.7.3
Steps to produce issue:
`
git clone https://github.com/b1ngx/nativewind-issue.git
cd nativewind-issue
npm i
mkdir android\app\src\main\assets
npx react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res
`
The output error:
`
D:\github\nativewind-issue>npx react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res
(node:15872) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
                Welcome to Metro v0.81.0
              Fast - Scalable - Integrated


error SHA-1 for file D:\github\nativewind-issue\node_modules\react-native-css-interop\.cache/android.js is not computed.
         Potential causes:
           1) You have symlinks in your project - watchman does not follow symlinks.
           2) Check `blockList` in your metro.config.js and make sure it isn't excluding the file path.
ReferenceError: SHA-1 for file D:\github\nativewind-issue\node_modules\react-native-css-interop\.cache/android.js is not computed.
         Potential causes:
           1) You have symlinks in your project - watchman does not follow symlinks.
           2) Check `blockList` in your metro.config.js and make sure it isn't excluding the file path.
    at DependencyGraph.getSha1 (D:\github\nativewind-issue\node_modules\metro\src\node-haste\DependencyGraph.js:181:13)
    at Transformer._getSha1 (D:\github\nativewind-issue\node_modules\metro\src\Bundler.js:15:26)
    at Transformer.transformFile (D:\github\nativewind-issue\node_modules\metro\src\DeltaBundler\Transformer.js:92:19)
    at Bundler.transformFile (D:\github\nativewind-issue\node_modules\metro\src\Bundler.js:43:30)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.transform (D:\github\nativewind-issue\node_modules\metro\src\lib\transformHelpers.js:134:12)
    at async transform (D:\github\nativewind-issue\node_modules\metro\src\DeltaBundler\Graph.js:164:26)
    at async visit (D:\github\nativewind-issue\node_modules\metro\src\DeltaBundler\buildSubgraph.js:82:29)
    at async Promise.all (index 3)
    at async visit (D:\github\nativewind-issue\node_modules\metro\src\DeltaBundler\buildSubgraph.js:92:5)
info Run CLI with --verbose flag for more details.
`

**Comment by bpn2k4** [comment]
Hi, i cloned @b1ngx repo [https://github.com/b1ngx/nativewind-issue.git](b1ngx/nativewind-issue) and encountered this error when using the bundle command.
OS: window 11
Node: v21.7.3
Steps to produce issue:
```bash
git clone https://github.com/b1ngx/nativewind-issue.git
cd nativewind-issue
npm i
mkdir android\app\src\main\assets
npx react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res
```
The output error:
```bash
D:\github\nativewind-issue>npx react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res
(node:15872) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
                Welcome to Metro v0.81.0
              Fast - Scalable - Integrated


error SHA-1 for file D:\github\nativewind-issue\node_modules\react-native-css-interop\.cache/android.js is not computed.
         Potential causes:
           1) You have symlinks in your project - watchman does not follow symlinks.
           2) Check `blockList` in your metro.config.js and make sure it isn't excluding the file path.
ReferenceError: SHA-1 for file D:\github\nativewind-issue\node_modules\react-native-css-interop\.cache/android.js is not computed.
         Potential causes:
           1) You have symlinks in your project - watchman does not follow symlinks.
           2) Check `blockList` in your metro.config.js and make sure it isn't excluding the file path.
    at DependencyGraph.getSha1 (D:\github\nativewind-issue\node_modules\metro\src\node-haste\DependencyGraph.js:181:13)
    at Transformer._getSha1 (D:\github\nativewind-issue\node_modules\metro\src\Bundler.js:15:26)
    at Transformer.transformFile (D:\github\nativewind-issue\node_modules\metro\src\DeltaBundler\Transformer.js:92:19)
    at Bundler.transformFile (D:\github\nativewind-issue\node_modules\metro\src\Bundler.js:43:30)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.transform (D:\github\nativewind-issue\node_modules\metro\src\lib\transformHelpers.js:134:12)
    at async transform (D:\github\nativewind-issue\node_modules\metro\src\DeltaBundler\Graph.js:164:26)
    at async visit (D:\github\nativewind-issue\node_modules\metro\src\DeltaBundler\buildSubgraph.js:82:29)
    at async Promise.all (index 3)
    at async visit (D:\github\nativewind-issue\node_modules\metro\src\DeltaBundler\buildSubgraph.js:92:5)
info Run CLI with --verbose flag for more details.
```

**Comment by b1ngx** [comment]
I created a github actions reproduce the issue

https://github.com/b1ngx/nativewind-issue/actions/runs/11590533057/job/32268306125#step:6:90

**Comment by marklawlor** [comment]
Can someone please test `nativewind@0.0.0-20241031002802`. I believe I've found the cause of the issue

**Comment by b1ngx** [comment]
Yes, it's fixed. 

[![build](https://github.com/b1ngx/nativewind-issue/actions/workflows/build.yml/badge.svg)](https://github.com/b1ngx/nativewind-issue/actions/workflows/build.yml)

**Comment by b1ngx** [comment]
> I've created github actions to reproduce the issue.
> 
> [![build](https://github.com/b1ngx/nativewind-issue/actions/workflows/build.yml/badge.svg)](https://github.com/b1ngx/nativewind-issue/actions/workflows/build.yml)
> 
> https://github.com/b1ngx/nativewind-issue/actions/runs/11590533057/job/32268306125#step:6:90

Yes, it's fixed.

**Comment by TCGXR** [comment]
Works for me as well! THX!!!!!

**Comment by marklawlor** [comment]
Fix published in `4.1.22`.

If you still get an error message, then you have a different issue that nobody has reported yet. Please create a new issue with a reproduction and instructions.
